activate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+dotfiles_dir=$(cd "$(dirname "$0")"; pwd)
+
+for name in vim vimrc gvimrc vimrc.bundles; do
+  if [ -e "${HOME}/.${name}" ]; then
```

Comment:
The `-f` ignores nonexistent files, so I think you can get rid of the `if` check.